src/Debugging/Drivers/SystemLogDebugger.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Saloon\Debugging\Drivers;
+
+use Saloon\Debugging\DebugData;
+
+class SystemLogDebugger extends DebuggingDriver
+{
+    public function name(): string
+    {
+        return 'syslog';
+    }
+
+    /**
+     * @param \Saloon\Debugging\DebugData $data
+     *
+     * @return void
+     * @throws \JsonException
+     */
+    public function send(DebugData $data): void
+    {
+        $encoded = json_encode($this->formatData($data), JSON_THROW_ON_ERROR);
+
+        syslog(LOG_DEBUG, print_r($encoded, true));
```

Comment:
`json_encode()` already turns it into a string, so no need for `print_r()`.

```suggestion
        syslog(LOG_DEBUG, $encoded);
```